src/main/java/me/moodcat/database/controllers/ChatDAO.java

```diff
@@ -0,0 +1,26 @@
+package me.moodcat.database.controllers;
+
+import static me.moodcat.database.entities.QChatMessage.chatMessage;
+
+import java.util.List;
+
+import javax.persistence.EntityManager;
+
+import me.moodcat.database.entities.ChatMessage;
+import me.moodcat.database.entities.Room;
+
+public class ChatDAO extends AbstractDAO<ChatMessage> {
+
+    private static final long NUMBER_OF_CHAT_MESSAGE = 10;
+
+    protected ChatDAO(final EntityManager entityManager) {
+        super(entityManager);
+    }
+
+    public List<ChatMessage> listByRoomId(final Room roomId) {
```

Comment:
So we really want to persist the messages or do we just store them in memory? We might want to consider the last option because hypothetically the amount of messages really adds up and we're only interested in the last few per room. (If we are really a chatting service). But this is only relevant for performance.